README.md

```diff
@@ -389,6 +392,48 @@ Uses the same options as [`createTag`](#createtagid-opts).
 
 Deletes the tag with the provided id and returns `undefined`. Note that this deletes the tag even if it is still attached to entries or assets.
 
+#### `setTagsForEntries(config)`
+
+For the given content type, updates the tags that are attached to its entries according to the user-provided `setTagsForEntry` function. For each entry, the CLI will call this function once, passing in the `from` fields, link objects of all tags that already are attached to the entry and link objects of all tags available in the environment. The `setTagsForEntry` function is expected to return an array with link objects for all tags that are to be added to the entry. If it returns `undefined`, the entry will be left untouched.
+
+**`config : Object`** – Content transformation definition, with the following properties:
+
+- **`contentType : string`** _(required)_ – Content type ID
+- **`from : array`** _(required)_ – Array of the source field IDs
+- **`setTagsForEntry : function (entryFields, entryTags, apiTags): object`** _(required)_ – Transformation function to be applied.
+    - `entryFields` is an object containing each of the `from` fields.
+    - `entryTags` is an array containing link objects of all tags
+already attached to the entry.
+    - `apiTags` is an array containing link objects of all tags
+available in the environment.
+
+##### `setTagsForEntries` Example
+
+``` javascript
+migration.createTag('department-sf').name('Department: San Francisco')
+migration.createTag('department-ldn').name('Department: London')
+
+const departmentMapping = {
+  'san-francisco': 'department-sf',
+  'london': 'department-ldn'
+}
+
+migration.setTagsForEntries({
+  contentType: 'news-article',
+  from: ['department'],
+  setTagsForEntry: (entryFields: any, entryTags: tagLink[], apiTags tagLink[]): tagLink[] => {
```

Comment:
The type should be capitalized: `TagLink`.
Also you forgot ` :` after `apiTags`.